### PR TITLE
✨ Add Card component 

### LINKED ISSUE #1: Creating basic Dashboard
Structure:
- [x] Header
- [x] Main
- [x] Footer

Components:
- [x] Burger-Menu
- [x] Headlines for main
- [x] Card-elements (work/private board, mood survey)

### PR DESCRIPTION
**Pull request by Naasgul**
This PR closes #1

### CODE REVIEW

**Review by Sina-Sche — APPROVED** [review]
👍 lgtm

File: client/src/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+        sed diam voluptua.{" "}
+      </p>
+    </BoardCard>
+  );
+};
+
+Card.propTypes = {
+  /**
+   * Hier die Auflistung der Farben, damit Storybook nicht abfuckt
```
Comment (ncbecker) [inline client/src/components/Card/Card.js:47]: 😂

**Review by ncbecker — APPROVED** [review]
lgtm 🚀

**Review by mhstrkmp — APPROVED** [review]
lgtm 🚀

**Pull request merged**